overlay/ProximityOneViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ProximityOneViewController.swift
+//  overlay
+//
+//  Created by Michelle Harvey on 3/16/16.
+//  Copyright © 2016 overlay. All rights reserved.
+//
+
+import UIKit
+
+class ProximityOneViewController: UIViewController {
```

Comment:
I think it is already? 

![image](https://cloud.githubusercontent.com/assets/5547897/13836025/94801962-ebba-11e5-9688-719b84aee7bf.png)